packages/connect/extension-resolver.cjs

```diff
@@ -0,0 +1,29 @@
+let defaultResolver;
+
+function requireDefaultResolver() {
+  if (!defaultResolver) {
+    try {
+      defaultResolver = import (`jest-resolve/build/defaultResolver`);
+    } catch (error) {
+      defaultResolver = import (`jest-resolve/build/default_resolver`);
```

Comment:
This is here to support older versions of jest .. we only need the first import and no try / catch